src/datachain/lib/convert/flatten.py

```diff
@@ -10,19 +10,25 @@
     Boolean,
     DateTime,
     Float,
+    Float32,
+    Float64,
```

Comment:
Does this close https://github.com/iterative/datachain/issues/12 ?

Reply:
If it does, please don't forget to rollback this change: https://github.com/iterative/datachain/pull/13.

Reply:
This doesn't close #12, but it will be easy to fix, I'll make separate PR for that.